src/app/article/markdown.pipe.ts

```diff
@@ -1,9 +1,9 @@
 import { Pipe, PipeTransform } from '@angular/core';
-import * as marked from 'marked';
 
 @Pipe({name: 'markdown'})
 export class MarkdownPipe implements PipeTransform {
-  transform(content: string): string {
+  async transform(content: string): Promise<string> {
+    const marked = await import('marked');
```

Comment:
Any advantage behind doing `import('marked')` inside `transform` method?

Reply:
It splits the marked module into its own chunk so that it’s only loaded if necessary.